charts/posthog/values.yaml

```diff
@@ -474,7 +474,15 @@ pluginsIngestion:
     maxUnavailable: 25%
 
   # -- Additional env variables to inject into the plugin-server stack deployment.
-  env: []
+  env:
+    - name: INGESTION_OVERFLOW_ENABLED
+      # -- Whether or not to send events to a "slow-lane" to be consumed
+      # separately by the pluginsIngestionOverflow deployment. When setting this
+      # to true, you much set pluginsIngestionOverflow.enabled to `true``
```

Comment:
nit: typo

```suggestion
      # to true, you must set pluginsIngestionOverflow.enabled to `true``
```

Reply:
Isn't this not a "must" if we enable both `ingestion` and `ingestionOverflow` in the same deployment? Not for us, but thinking about hobby deployments that want to try out overflow (although may be a stretch).